pkg/build/api/validation/validation_test.go

```diff
@@ -67,7 +63,7 @@ func TestBuildValidationFailure(t *testing.T) {
 			Phase: buildapi.BuildPhaseNew,
 		},
 	}
-	if result := ValidateBuild(build); len(result) != 3 {
+	if result := ValidateBuild(build); len(result) != 2 {
```

Comment:
I'm convinced it's a bad test. I had a look at the body of `TestBuildValidationFailure`, it's hard to tell what validation aspect it is trying to test, and all it verifies is that there are N errors. Maybe whatever validation errors we're covering here is covered elsewhere in a table-driven test.